client/app/components/RatingsAndReviews/ReviewList/ReviewButtonsContainer/AddReviewButton/ReviewCharacteristics/ReviewCharacteristic.jsx

```diff
@@ -2,86 +2,134 @@
 import React, { useState } from 'react';
 
 const ReviewCharacteristic = ({ name, id, reviewCharsObj, setReviewCharsObj }) => {
-  const [sizeCharacteristic, setSizeCharacteristic] = useState(0);
-  const [widthCharacteristic, setWidthCharacteristic] = useState(0);
-  const [comfortCharacteristic, setComfortCharacteristic] = useState(0);
-  const [qualityCharacteristic, setQualityCharacteristic] = useState(0);
-  const [lengthCharacteristic, setLengthCharacteristic] = useState(0);
-  const [fitCharacteristic, setFitCharacteristic] = useState(0);
+  // const [sizeCharacteristic, setSizeCharacteristic] = useState(0);
+  // const [widthCharacteristic, setWidthCharacteristic] = useState(0);
+  // const [comfortCharacteristic, setComfortCharacteristic] = useState(0);
+  // const [qualityCharacteristic, setQualityCharacteristic] = useState(0);
+  // const [lengthCharacteristic, setLengthCharacteristic] = useState(0);
+  // const [fitCharacteristic, setFitCharacteristic] = useState(0);
   const [rowCharSelection, setRowCharSelection] = useState(0);
 
   const handleSelection = (e) => {
     setRowCharSelection(e.target.value);
+
+    // update parent state
     setReviewCharsObj({ ...reviewCharsObj, [id]: Number(e.target.value) });
   };
 
+  const charObject = {};
```

Comment:
Char object is a little confusing of a name and doesn't convey anything to tell me what it actually holds. Again, this wouldn't fail you a code review, but it is something to keep in mind